tests/test_warpedPsf.cc

```diff
@@ -247,24 +309,23 @@ BOOST_AUTO_TEST_CASE(warpedPsf)
     unwarped_psf->evalABC(a, b, c, q);
 
     Eigen::Matrix2d m0;
-    m0 << a, b,
-          b, c;
+    m0 << a, b, b, c;
```

Comment:
I assume this is to make clang-format happy?  Is there a way to do that and keep it obvious that these are entries in a 2x2 matrix like before?

Reply:
Sorry, I just blindly ran `clang-format`. Since I'm not familiar enough with Eigen to know if there's another way to write this, I'll just revert.

Reply:
Aha, there's a workaround: https://clang.llvm.org/docs/ClangFormatStyleOptions.html#disabling-formatting-on-a-piece-of-code